src/plugins/home/server/lib/tutorial_schema.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Joi from 'joi';
+
+const PARAM_TYPES = {
+  NUMBER: 'number',
+  STRING: 'string',
+};
+
+const TUTORIAL_CATEGORY = {
+  LOGGING: 'logging',
+  SIEM: 'siem',
+  METRICS: 'metrics',
+  OTHER: 'other',
+};
+
+const dashboardSchema = Joi.object({
+  id: Joi.string().required(), // Dashboard saved object id
+  linkLabel: Joi.string().when('isOverview', {
+    is: true,
+    then: Joi.required(),
+  }),
+  // Is this an Overview / Entry Point dashboard?
+  isOverview: Joi.boolean().required(),
+});
```

Comment:
Is there a reason to be using `joi` directly instead of `@kbn/config-schema`?

Reply:
That's the way it was prior to the move and I didn't want to introduce too many changes at once.

Reply:
Ok, thanks for the answer. Not everything in `joi` is available in `config-schema`, so migration may be not that obvious. Will try to look at the migration PR. `joi` is fine for now.